docs/index.md

```diff
@@ -1,10 +1,130 @@
 OSG Site Administrator Documentation
 ====================================
 
-!!! note
-    The documentation contained here is currently a work in progress as we migrate our TWiki documentation to GitHub. Please let us know of any issues that you encounter by [opening a ticket](https://ticket.opensciencegrid.org) or e-mailing [goc@opensciencegrid.org](mailto:goc@opensciencegrid.org). Please identify errors, inconsistencies, or incorrect information in the documentation, and also suggest extra content that you feel would be helpful to include in the documentation.
-
-Welcome to the home page of the Open Science Grid (OSG) Site Administrator documentation! If you're a new OSG site adminstrator, start with the [Installation Overview](install-overview). If you are not a site adminstrator...
+Welcome to the home page of the Open Science Grid (OSG) Site Administrator documentation! If you are not a site adminstrator...
 
 - If you are a researcher interested in using OSG resources, you can find user documentation [here](https://support.opensciencegrid.org/support/home). 
 - If you'd like to learn more about the OSG and our mission, visit our website [here](https://www.opensciencegrid.org/).
+
+This document outlines the overall installation process for an OSG site and provides many links into detailed installation, configuration, troubleshooting, and similar pages. If you do not see software-related technical documentation listed here, try the search bar to the left or contacting us at [goc@opensciencegrid.org](mailto:goc@opensciencegrid.org).
+
+Plan the Site
+-------------
+
+If you have not done so already, [plan the overall architecture of your OSG site](site-planning). It is recommended that your plan be sufficiently detailed to include the OSG hosts that are needed and the main software components for each host. Be sure to consider [the operating systems that OSG supports](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SupportedOperatingSystems). A simple way to organize this information is in a table; for example, a basic site might include:
```

Comment:
> A simple way to organize this information is in a table

Maybe I'm just overly sensitive, but this seems borderline insulting of the reader's intelligence.

I don't think the sentence loses content if we just have it be:

> For example, a basic site might include: